### PR TITLE
Feature/event tagging framework typescripts for event api

### PR DESCRIPTION
**Pull request by horridparlor**

### CODE REVIEW

**Comment by anhminh10a2hoa** [comment]
remember to run `npm run format` in the client folder

File: client/src/components/Cards/EventCard.tsx [hunk 1]
```diff
@@ -1,55 +1,58 @@
...
-        return EventTagType.MUSIC
-      case 4:
-        return EventTagType.OUTDOOR
-      case 5:
-        return EventTagType.RELIGION
-      case 6:
-        return EventTagType.SPORTS
-      default:
-        return EventTagType.DEFAULT
+const EventCard: React.FC = (params: { event: EventObj} ) => {
```
Comment (anhminh10a2hoa) [inline client/src/components/Cards/EventCard.tsx:16]: it should be `const EventCard: React.FC<EventCardProps> = ({ event }: EventCardProps) => {`

File: client/src/components/Cards/EventCard.tsx [hunk 2]
```diff
@@ -1,55 +1,58 @@
...
+    const { language } = useLocation();
+    const {t} = useTranslation();
+    const {event} = params;
+    const [dayTemp, setDay] = useState('0');
```
Comment (anhminh10a2hoa) [inline client/src/components/Cards/EventCard.tsx:20]: `const dayTemp = String(new Date(event.startTime).getUTCDate()).padStart(2, '0')`

**Review by anhminh10a2hoa — CHANGES_REQUESTED** [review]
remember to run `npm run format` in the client folder

**Comment by horridparlor** [comment]
Is everything good now?

**Review by anhminh10a2hoa — APPROVED** [review]
Looks good! I will approve it

**Pull request merged**